Assets/Scripts/GUI/PolyPanel.cs

```diff
@@ -64,6 +65,7 @@ public class PolyPanel : ModalPanel {
     m_NoLikesMessage.SetActive(false);
     m_NotLoggedInMessage.SetActive(false);
     m_OutOfDateMessage.SetActive(false);
+    m_NotSupportedMessage.SetActive(false);
```

Comment:
The actual code changes are gated behind platform defines, I can only wonder if it's giving a NRE when trying to disable here.

Reply:
This should have broken other desktop builds if so, how are the others?

Reply:
Is there a way I can see logs?

Reply:
not without doing a dev build i think - i'll give it a test tonight

Reply:
No idea, OS X is my only platform (and Q2). I'll try a build too, but won't be able to run until Sat. night.

If it helps, it hangs and CPU shoots way up, but there's no crash or exit

Reply:
Ok, I did manage to get a build in before I sign off. This one broke it; reverting it against the tip fixes it. Thanks

Reply:
Okay, easy enough fix to make it nullable! Thanks for checking

Reply:
Just looking into this and found the same issue.

Reply:
I can confirm that #91 fixes it